src/screens/Ticketing/Tickets/TicketReservation.tsx

```diff
@@ -128,6 +129,11 @@ const useStyles = StyleSheet.createThemeHook((theme) => ({
   ticketInfoContainer: {
     padding: theme.spacings.medium,
   },
+  reservationStatus: {
+    flex: 1,
+    textAlign: 'right',
+    marginLeft: theme.spacings.xLarge,
```

Comment:
xLarge sounds a bit much here. The padding doesn't look very consistent in the sketches, but i think 8px (small) looks the most correct. 

![Screenshot 2022-09-22 at 12 06 05](https://user-images.githubusercontent.com/1774972/191719859-db4b7306-a905-4c12-aa78-adad2d7df67d.png)
![Screenshot 2022-09-22 at 12 05 47](https://user-images.githubusercontent.com/1774972/191719862-2f624221-c728-466c-a174-dfc6237c0f0e.png)
![Screenshot 2022-09-22 at 12 05 38](https://user-images.githubusercontent.com/1774972/191719864-fbcd038f-7ab5-4e6a-ab6a-fbb7591bcd3d.png)

```suggestion
    marginLeft: theme.spacings.small,
```

Reply:
I think the padding is xlarge in farecontract as well 🤔

Reply:
but I agree, it does look big. 👍